TestFoundation/TestNSOperationQueue.swift

```diff
@@ -36,17 +41,21 @@ class TestNSOperationQueue : XCTestCase {
 
     func test_OperationPriorities() {
         var msgOperations = [String]()
-        let operation1 : BlockOperation = BlockOperation(block: {
+        let operation1 = BlockOperation(block: {
             msgOperations.append("Operation1 executed")
+            sleep(1)
```

Comment:
Sorry - sleep cannot be a fix for issues like this. Instead we should use a semaphore to signal completion of whatever work we are waiting on.